src/services/utils.ts

```diff
@@ -80,3 +121,22 @@ const parseError = (text: string) => {
         return null;
     }
 };
+
+export const getRequestParamFromList = (
+    params: string[] = [],
+    paramName: string
+) => {
+    return new URLSearchParams(params.map((param) => [paramName, param]));
+};
+
+export const getWsBase = () =>
+    document.baseURI
+        .replace(/^http:\/\//, 'ws://')
+        .replace(/^https:\/\//, 'wss://');
+
+export const FetchStatus = {
+    SUCCEED: 'SUCCEED',
+    FAILED: 'FAILED',
+    IDLE: 'IDLE',
+    RUNNING: 'RUNNING',
+};
```

Comment:
It has nothing to do here, as it's an enum for components, not for api.

Reply:
True, wasn't worth the pain to move in the end, I reverted it